the-devastating-chickens/BangazonSite
# Av add to order redux

# Description
When the customer inspects the details page of a product and clicks the "Add to Order" button, the product is added to their current order. If no current order exists, an order is automatically created and the product is added to it.

The order details page was generated with code to display a list of orders;. This is placeholder code simply for testing purposes. Whoever does the ticket for the the order details page should overwrite it as per the ticket story.

Fixes Issue # 34

## Type of change New Feature

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout av-add-to-order-redux
navigate to product categories
click on a product
click 'add to order'
an order will be generated and you will be taken to your order page
you will see the product listed at the bottom
go back to product categories
add another item to cart (preferably a different item)
since there's now an active order, that order should be the one the new item is added to, and there will be two items in the list

# Checklist:
